topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+---
+layout: tutorial_hands_on
+
+title: "Reproducing Critical Assessment of Metagenome Interpretation assembly challenge on marine dataset with Galaxy"
+zenodo_link: ""
+questions:
+  - "How to reproduce CAMI challenge?"
+  - "How to add/update tool in Galaxy?"
+  - "How to benchmark results of CAMI challenge?"
+objectives:
+  - "Be familiar with CAMI challenge"
+  - "Be able to select one challenge and benchmarking datasets"
+  - "Be able to upload benchmarking datasets into Galaxy"
+  - "Be familiar with Galaxy computational resources"
+  - "Be familiar with existing tools in Galaxy"
+  - "Be familiar with databases / reference genomes that are available in Galaxy"
+  - "Be able to add/update the tool in Galaxy"
+  - "Be able to choose assembling tool based on dataset metadata"
+  - "Perform assembly with Flye, Megahit, Abyss, MetaSPAdes tools"
+  - "Produce a benchmarking analysis of these assemblies with Quast, Bowtie2, Map with Minimap2, Samtools, and MultiQC tools"
+  - "Create plots with Python to compare results"
+time_estimation: "40H00M"
```

Comment:
Thank you for the comment. Do you have any advice how can I count this number?

Reply:
It's usually "how long does it take to teach/follow this tutorial from start to finish", if some of the assembly steps take that long, then, that can be accurate!

Reply:
> It's usually "how long does it take to teach/follow this tutorial from start to finish", if some of the assembly steps take that long, then, that can be accurate! I was just surprised at the big number :)

Then I would put more since the slowest assembler took 41h and after that Quast should be run and it took 3h. And it was just the pure time of one assembler (out of 9) and Quast (there were other tools run though).